treatment_index.py

```diff
@@ -367,22 +355,351 @@ def create_treatment_points(schema, sde_file):
 
 
 
-def rename_treatment_points(schema, sde_file, connection):
+def rename_treatment_points(schema, sde_file, cursor, treatment_index):
 
     # rename current table to backup
-    rename_postgres_table(connection, schema, 'treatment_index_facts_nfpors_points', 'treatment_index_facts_nfpors_points_backup')
-    logger.info(f'{schema}.treatment_index_facts_nfpors_points renamed to {schema}.treatment_index_facts_nfpors_points_backup')
+    rename_postgres_table(cursor, schema, f'{treatment_index}_points', f'{treatment_index}_points_backup')
+    logger.info(f'{schema}.{treatment_index}_points renamed to {schema}.{treatment_index}_points_backup')
 
     # rename temp to current table
-    rename_postgres_table(connection, schema, 'treatment_index_facts_nfpors_points_temp', 'treatment_index_facts_nfpors_points')
-    logger.info(f'{schema}.treatment_index_facts_nfpors_points_temp renamed to {schema}.treatment_index_facts_nfpors_points')
+    rename_postgres_table(cursor, schema, f'{treatment_index}_points_temp', f'{treatment_index}_points')
+    logger.info(f'{schema}.{treatment_index}_points_temp renamed to {schema}.{treatment_index}_points')
 
     # rename backup to temp so we can delete it
-    rename_postgres_table(connection, schema, 'treatment_index_facts_nfpors_points_backup', 'treatment_index_facts_nfpors_points_temp')
-    logger.info(f'{schema}.treatment_index_facts_nfpors_points_backup renamed to {schema}.treatment_index_facts_nfpors_points_temp')
+    rename_postgres_table(cursor, schema, f'{treatment_index}_points_backup', f'{treatment_index}_points_temp')
+    logger.info(f'{schema}.{treatment_index}_points_backup renamed to {schema}.{treatment_index}_points_temp')
 
     # clean up so we can create points again later
-    arcpy.management.Delete(os.path.join(sde_file, f"sweri.{schema}.treatment_index_facts_nfpors_points_temp"))
+    arcpy.management.Delete(os.path.join(sde_file, f"sweri.{schema}.{treatment_index}_points_temp"))
+
+# BEGIN Common Attributes Functions
+
+def add_fields_and_indexes(feature_class, region):
+    arcpy.management.AddField(feature_class,'included', 'TEXT')
```

Comment:
can likely simplify this by creating a list of tuples with your index info and just iterating over it or the like

Reply:
This has been updated